build/Makefile

```diff
@@ -489,7 +489,7 @@ build-extensions-binary-linux-arm64: $(ensure-build-image)
 lint: LINT_TIMEOUT ?= 15m
 lint: $(ensure-build-image)
 	docker run -t -e "TERM=xterm-256color" -e "$(gomod_on)" --rm $(common_mounts) -w $(workdir_path) $(DOCKER_RUN_ARGS) $(build_tag) bash -c \
-		"golangci-lint run ./examples/... && golangci-lint run --timeout $(LINT_TIMEOUT) ./..."
+		"golangci-lint run ./examples/... && golangci-lint run --timeout $(LINT_TIMEOUT) ./... && find ../ -name '*.yml' -o -name '*.yaml' | grep -v './install/' | xargs yamllint"
```

Comment:
two things here:
1. Are you sure about `..`? I think you're at the top of the repo here
2. I think we should prefer exclusions using the yamllint config itself rather than `find`. You can just use `yamllint .` and then [ignore paths in a `.gitignore` style way](https://yamllint.readthedocs.io/en/stable/configuration.html#ignoring-paths). That allows people to just run `yamllint` outside docker or the Makefile and keep the config the same. I think it would just look roughly like:
```
ignore:
  - /install/
```

I wouldn't bother configuring the yaml suffixes, it looks like the default is appropriate.